src/extension/tab/helpers.ts

```diff
@@ -45,17 +77,39 @@ export function getQueries(
       if (name === "IntrospectionQuery") {
         return;
       }
+
+      const { pollingInfo } = observableQuery;
+      const { networkStatus, error } = observableQuery.getCurrentResult(false);
+
       queries.push({
         document,
         variables,
         cachedData: diff.result,
         options: getQueryOptions(oc),
+        networkStatus,
+        error: error ? serializeApolloError(error) : undefined,
+        pollInterval: pollingInfo && Math.floor(pollingInfo.interval),
       });
     });
   }
   return queries;
 }
 
+function serializeApolloError(error: ApolloError): SerializedApolloError {
+  return {
+    clientErrors: error.clientErrors.map((e) => e.message),
+    name: "ApolloError",
+    networkError: error.networkError
+      ? serializeError(error.networkError)
+      : undefined,
+    message: error.message,
+    graphQLErrors: error.graphQLErrors.map((e) =>
+      e.constructor.name === "GraphQLError" ? e.toJSON() : e
```

Comment:
This should never be the case with Apollo Client internal code, it could only happen if a user-written `ApolloLink` somehow injects a `GraphQLError` instance. In that case, `JSON.stringify` would call `toJSON` anyways.
```suggestion
      e
```